.travis.yml

```diff
@@ -4,32 +4,61 @@ cache: pip
 matrix:
   include:
     - name: "Python 2.7 on Linux"
+      env: NODE_GYP_FORCE_PYTHON=python2
```

Comment:
On Linux runs Travis will set the environment variable $TRAVIS_PYTHON_VERSION == 2.7 so perhaps that it would be better to leverage that rather than create a new environment variable.

Reply:
As I think about it more, I now understand that you are looking for a more general solution that is useful beyond just Travis CI.  What if we name the variable __NODE_GYP_FORCE_PYTHON2__ and set it to True in legacy Python environments and set it to False or leave it undefined in Python 3 environments?  In this way we would not need to carry this baggage in future years.